lib/elements/text.js

```diff
@@ -4,10 +4,14 @@ const { cursor } = require('sisteransi');
 const { style: stl, clear } = require('../util');
 
 class TextPrompt extends Prompt {
-  constructor({ message, cursor, initial, style = 'default' }) {
+  constructor({ defaulter, message, cursor, initial, style = 'default' }) {
```

Comment:
That's one of the reasons why i don't use destructuring so much in that way, but more Object.assign.

So it can be changed to something like that

```js
constructor(options) {
  const opts = Object.assign({ style: 'default' }, options)
  const { message, cursor, initial } = opts

  // use `opts.default` later
}
```